README.md

```diff
@@ -161,12 +161,18 @@ If you need to ignore specific translation keys, follow this process:
 
 ### Setup mailing
 
-1. Go to the production app in the Heroku dashboard.
-2. Add Sendgrid addon.
-3. Click the added Sendgrid resource, go through the confirmation steps.
-4. Set up DKIM on the client's domain (under sender authentication -> domain authentication).
-5. Go to staging and repeat the above steps.
-6. If you want the review apps to send emails: Copy the staging app's Sendgrid credentials and add them to the review apps config vars. Do not add the Sendgrid addon to review apps in `app.json` (Sendgrid will ban our account).
+1. Register and add to 1Password new sendinblue account
+   1. Register: https://app.sendinblue.com/account/register
+   2. Email: Client google group account
+   3. Password: Generate in 1Password
+2. You must validate a phone number in order to start sending mails.
```

Comment:
I wonder if we can standardize this more, e.g. have a specific phone that we always use.

Reply:
@hcarreras What is your input on this?

Reply:
Totally agree. I'll ask Angelle to get us a shared phone number